packages/react-ui-components/src/components/SchemaForm/Widgets/BaseInput/BaseInput.tsx

```diff
@@ -0,0 +1,42 @@
+import { EInputFieldType, EValidationState } from '@kelvininc/ui-components';
+import classNames from 'classnames';
+import { isArray, isEmpty } from 'lodash-es';
+import React, { useMemo } from 'react';
+import { KvTextField } from '../../../stencil-generated';
+import styles from './BaseInput.module.scss';
+import { WidgetProps } from '@rjsf/core';
+import { INPUT_TYPES } from './BaseInput.config';
+import { JSONSchema7TypeName } from 'json-schema';
+
+const getInputType = (type?: JSONSchema7TypeName | JSONSchema7TypeName[]) => (type && !isArray(type) ? INPUT_TYPES[type] ?? EInputFieldType.Text : EInputFieldType.Text);
+
+const BaseInput = ({ id, placeholder, required, readonly, disabled, label, value, onChange, onBlur, autofocus, options, schema, rawErrors = [], uiSchema, type }: WidgetProps) => {
+	const _onChange = (value: CustomEvent<string>) => onChange(value?.detail ? value.detail : options.emptyValue);
+	const _onBlur = (value: CustomEvent<string>) => onBlur(id, value.detail);
+	const inputType = useMemo(() => type || getInputType(schema.type), [type, schema.type]);
```

Comment:
Seems to me that you should use the "nullish coalescing" operator here instead of the logical operator "or".

Reply:
I resolve that on next PR.